check-code-compliance/action.yaml

```
@@ -168,7 +169,7 @@ runs:
         JAVASCRIPT_DEFAULT_STYLE: prettier
         DISABLE_ERRORS: "true"
     - name: Archive mega linter artifacts
-      if: ${{ success() }} || ${{ failure() }}
+      if: (${{ success() }} || ${{ failure() }}) && ${{ github.event_name }} == 'pull_request'
```

```suggestion
      if: (success() || failure()) && github.event_name == 'pull_request'
```

Can't we just omit the first part of the expression `(success() || failure())`? Or exists a third state?

---

This is to upload artifacts even when megalint failed.